test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -259,14 +295,15 @@ def run_test(self, model, input, rtol=1e-3, atol=1e-7, do_constant_folding=True,
                  batch_size=2, use_gpu=True, dynamic_axes=None, test_with_inputs=None,
                  input_names=None, output_names=None, fixed_batch_size=False, dict_check=True,
                  training=None, remained_onnx_input_idx=None):
-        def _run_test(m, remained_onnx_input_idx):
+        def _run_test(m, remained_onnx_input_idx, flatten=True):
```

Comment:
this `flatten` flag seem not used anywhere, is this needed?

Reply:
It is used in line 175, maybe a rebase issue.